src/models/launch_1/customization.md

```diff
@@ -0,0 +1,95 @@
+
+**Tools required:** MX-style switch puller  
+**Time estimate:** ~1 minute per switch  
+**Difficulty:** Medium <span style="color:orange;">●</span>  
+
+Begin by [removing the keycap](#removing-and-installing-keycaps) on the switch you wish to remove. Place the two tongs of the switch removal tool onto the switch's locking tabs on the top and bottom edges of the switch. The hooks should squeeze the white tabs inwards and hook under the switch's clear plastic top housing. Pinch the tongs together a little to help release the switch's locking clips from the cutout in the top case, and pull the switch upwards out of the socket. The switch's locking tabs can sometimes bind on the edges of the top case cutouts, so if the switch feels stuck, remove the tool and press the switch back down into place, then try removing it again. Switches take more force to remove than keycaps. 
+
+![Removing a switch](./img/switch-removal.jpg)
+
+When installing a switch, it is important to first make sure the switch's pins are straight. The pins have to plug into the socket, and they'll miss their slots if they are bent.
+
+![Straighten switch pins](./img/switch-install-pins-straight.jpg)
+
+The switch should be oriented so that the clear LED shine-through window is towards the top of the socket, where the LED is located on the PCB.
+
+![Switch orientation](./img/switch-install-orientation.jpg)
+
+Place the switch straight into the plate. You should be able to feel whether the pins are plugging into the socket correctly. 
+
+**Note:** Forcing the pins into the hot swap socket could damage the PCB. If you feel excessive resistance, pull the switch out, double-check that the pins are straight, and try installing the switch again.
+
+Once you feel the pins plugging into the hot swap socket, press the switch down with more force to snap it into place in the chassis. 
+
+![Installing a switch](./img/switch-installation.jpg)
+
+## Removing the PCB
+
+**Tools required:** Keycap puller, MX-style switch puller, and cross-head (Phillips) screwdriver  
+**Time estimate:** 50 minutes  
+**Difficulty:** High <span style="color:red;">●</span>  
+
+If the PCB in your Launch ever becomes damaged, it can be replaced using these instructions. It is also recommended to completely remove the PCB if you are changing all of the switches on your Launch at the same time. This makes switch installation much easier and makes it much less likely to bend switch pins or damage the PCB.
+
+Begin by [removing all of the keycaps](#removing-and-installing-keycaps), and then [remove all of the switches](#removing-and-installing-switches). Next, turn the keyboard over and remove the eight screws that hold on the bottom panel and remove the bottom panel. Lifting the bottom panel out of the chassis may be easier with the magnetic lift bar installed.
+
+![Removing bottom panel screws](./img/bottom-panel-screws.jpg)
```

Comment:
We don't typically show the usage of a screwdriver on computer service manuals. It makes sense for the keycap picker because that's an unusual tool, but is it necessary here? A more head-on photo without having to hold the screwdriver there might make it easier to get good lighting on the screws.